e2e/features/address-manual.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Address details are entered manually without using postcode lookup
+
+  Background:
+    Given I am signed in as a Legal Rep
+    And I create a new case
+    And I complete the Home Office reference page
+    And I complete the Basic details page
+
+  @create-case @lodge-appeal@alternate @RIA-691
+  Scenario: Lodge an appeal application with a manually entered address
+
+    Given I am signed in as a Legal Rep
+    Given I create a new case
+    Given I complete the Home Office reference page
+    Given I complete the Basic details page
```

Comment:
These 4 steps have already been defined in the `Background:` section, so should be removed

Reply:
👍